addons/project/models/project_project.py

```diff
@@ -125,6 +129,7 @@ def _read_group_stage_ids(self, stages, domain, order):
         'resource.calendar', string='Working Time', compute='_compute_resource_calendar_id')
     type_ids = fields.Many2many('project.task.type', 'project_task_type_rel', 'project_id', 'type_id', string='Tasks Stages')
     task_count = fields.Integer(compute='_compute_task_count', string="Task Count")
+    closed_task_count = fields.Integer(compute='_compute_task_count', string="Closed Tasks Count")
```

Comment:
There is an inconsistence with the string between the field defined in `project.update` and this one. Also, for `task_count` is `Task Count` and this one is `Closed Tasks Count` and not `Closed Task Count`, is it expected? 🧐 🤔